### DISCUSSION TITLE
MAUI/UNO examples?

### DISCUSSION THREAD

**Discussion by marcusjm** [OrchardCMS/OrchardCore#10984] [Q&A]
Has anyone seen examples/videos of someone connecting a mobile/uno/maui app to OrchardCore?

**Comment by hishamco** [comment]
I don't see any, but it's good for OSS. OrchardCore Contrib (OCC) already planned to make a prototype for Blazor, if you want to add a MAUI app it will be great contribution

BTW OC admin already adaptive in the mobile screens, @Skrypt @agriffard and me already did a lot of PRs to make this happen

**Comment by marcusjm** [comment]
At the very least it should be a good way to handle the authentication part.

**Reply by hishamco** [reply]
OpenId module is already out of the box in case you wanna use to use Orchard Core from other UI framework

**Reply by marcusjm** [reply]
Yes but some samples/videos wouldn't be too bad I guess. Depends on time of course. At the same time, any marketing for Orchard Core would be good.
